test/test-aux.jl

```diff
@@ -0,0 +1,6 @@
+# file with auxiliary functions for the testing
+
+function _read_csv_folder(connection, input_dir)
+    schemas = TulipaEnergyModel.schema_per_file
+    return TulipaIO.read_csv_folder(connection, input_dir; schemas)
+end
```

Comment:
```suggestion
import TulipaIO

function read_csv_folder(connection, input_dir)
    schemas = TulipaEnergyModel.schema_per_file
    return TulipaIO.read_csv_folder(connection, input_dir; schemas)
end
```

I think if you do this you don't need to do the renames in `test-case-studies.jl`.  I'm a little unsure about the namespace, @abelsiqueira should be able to confirm.

You can also ignore this suggestion since it's not particularly important.

Reply:
That's true, I didn't think of that. It should work if `read_csv_folder` is not exported by TulipaIO and we don't  explicitly import it from TulipaIO.

Reply:
@suvayu, actually, I prefer renaming and having "_" at the beginning to make it clear that it is an internal function, and that is different from the TulipaIO one. It is clearer for no Julia experts/programmers because you don't need to think about the situation that Abel is describing.

Reply:
As I said, you can ignore my comment if you prefer.